### PR TITLE
Update time_entry.rst to fix doc issue 165

### PR DESCRIPTION
**Pull request by loechel**
Update time_entry.rst to fix doc issue #165, the hour param should not be only integer it should be integer or float, I do not see any limitation on that and Redmine takes also floats for the time entries. 
Addusted also doctests to test and reflect changes

### CODE REVIEW

**Comment by maxtepkeev** [comment]
Thanks. Actually Redmine accepts both integer and float for hour parameter. Can you please adjust your pull request for this. After that I will be happy to merge it.

**Comment by loechel** [comment]
adjusted to fix #165

File: docs/resources/time_entry.rst [hunk 1]
```diff
@@ -23,7 +23,7 @@ create
     Creates new time entry resource with given fields and saves it to the Redmine.
 
     :param integer issue_id or project_id: (required). The issue id or project id to log time on.
-    :param integer hours: (required). The number of spent hours.
+    :param integer or float hours: (required). The number of spent hours.
```
Comment (maxtepkeev) [inline docs/resources/time_entry.rst]: Won't work this way, have a look at the definition of spent_on param below

File: docs/resources/time_entry.rst [hunk 2]
```diff
@@ -157,7 +157,7 @@ update
 
     :param integer resource_id: (required). Time entry id.
     :param integer issue_id or project_id: (optional). The issue id or project id to log time on.
-    :param integer hours: (optional). The number of spent hours.
+    :param integer or float hours: (optional). The number of spent hours.
```
Comment (maxtepkeev) [inline docs/resources/time_entry.rst]: Won't work this way, have a look at the definition of spent_on param below

**Comment by maxtepkeev** [comment]
Yes, this is good, thank you. Will merge it a little bit later today.

**Pull request merged**